Source/JavaScriptCore/runtime/Structure.h

```diff
@@ -988,7 +988,7 @@ class Structure : public JSCell {
     bool isValid(JSGlobalObject*, StructureChain* cachedPrototypeChain, JSObject* base) const;
 
     // You have to hold the structure lock to do these.
-    JS_EXPORT_PRIVATE void pin(const AbstractLocker&, VM&, PropertyTable*);
```

Comment:
When `JS_EXPORT_PRIVATE` is attached, LTO even cannot inline it since this is explicitly exposed outside of this framework.

Reply:
Maybe you should add a comment here so that no one mistakenly adds `JS_EXPORT_PRIVATE` back in the future.